### PR TITLE
Translate common texts into Japanese

### PR DESCRIPTION
**Pull request by JohnTitor**
Similar to #901 but for Japanese.
@woodyZootopia Could you check?

### CODE REVIEW

**Comment by JohnTitor** [comment]
The notice says:
> It might be incomplete, outdated or contain errors. Please report any issues!

But Philipp cannot respond to them as mentioned in https://github.com/phil-opp/blog_os/issues/895#issuecomment-756677259 so I think it makes sense to have a tracking issue for the translation and link to it on the notice. @phil-opp Does it work for you?

**Comment by phil-opp** [comment]
Yes, seems like a good solution!

File: blog/config.toml [hunk 1]
```diff
@@ -74,16 +74,16 @@ word_separator = "and"
...
-translated_content_notice = "This is a community translation of the <strong><a href=\"_original.permalink_\">_original.title_</a></strong> post. It might be incomplete, outdated or contain errors. Please report any issues!"
-translated_by = "Translation by"
-word_separator = "and"
+toc = "目次"
+all_posts = "« すべての記事へ"
+comments = "コメント"
+comments_notice = "可能な限りコメントは英語で残すようにしてください。"
+readmore = "もっと読む »"
+not_translated = "（この記事はまだ翻訳されていません。）"
+translated_content = "この記事は翻訳されています："
```
Comment (woodyZootopia) [inline blog/config.toml:83]:
```suggestion
translated_content = "この記事は翻訳されたものです："
```
seems more natural?

**Comment by woodyZootopia** [comment]
I left a suggestion, otherwise it looks perfect!

**Comment by JohnTitor** [comment]
@phil-opp Added a link to #906 and ready to merge :)

**Comment by phil-opp** [comment]
Thank you!

**Comment by phil-opp** [comment]
If you like, you can also add a translation for the introduction text in https://github.com/phil-opp/blog_os/blob/master/blog/content/_index.ja.md

**Comment by woodyZootopia** [comment]
I'm on it!

**Comment by woodyZootopia** [comment]
#907

**Comment by hamidrezakp** [comment]
> The notice says:
> 
> > It might be incomplete, outdated or contain errors. Please report any issues!
> 
> But Philipp cannot respond to them as mentioned in [#895 (comment)](https://github.com/phil-opp/blog_os/issues/895#issuecomment-756677259) so I think it makes sense to have a tracking issue for the translation and link to it on the notice. @phil-opp Does it work for you?

Nice idea, we can have an issue for Persian translation.

**Pull request merged**